mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/building/impl/PojoIdentityMappingCollectorImpl.java

```diff
@@ -69,6 +77,7 @@ void closeOnFailure() {
 		try ( Closer<RuntimeException> closer = new Closer<>() ) {
 			closer.push( IdentifierMappingImplementor::close, identifierMapping );
 			closer.push( RoutingKeyProvider::close, routingKeyProvider );
+			closer.push( BeanHolder::close, providedIdentifierBridgeHolder );
```

Comment:
I don't think you need this, as `identifierMapping.close()` (above) will effectively close the bridge, and it's very unlikely (impossible?) that a failure occurs between the assignment of `providedIdentifierBridgeHolder` and the assignment of `identifierMapping`.

So, to clarify: you can remove `providedIdentifierBridgeHolder` and use a local variable instead, then remove this line.

Reply:
Yes, I also think that it is impossible.
I applied the changes, let's wait for the CI, then I will merge it.
Thanks